node_modules/gh-core/lib/db.js

```diff
@@ -353,11 +353,13 @@ var _setUpModel = function(sequelize) {
      * @property  {String}      displayName         The display name of the serie
      * @property  {String}      description         The description of the serie
      * @property  {String}      image               The path to an image for the serie
+     * @property  {Object}      metadata            The extra metadata of the serie
```

Comment:
Should this be a generic "whatever we want" field or should we limit it to just "summary" for now?